ecosystem/sep-0012.md

```diff
@@ -182,25 +196,38 @@ Name | Type | Description
-PROCESSING | KYC process is in flight and client can check again in the future to see if any further info is needed
+ACCEPTED | All required KYC fields have been accepted. It is possible for an accepted customer to move back to another status if the KYC provider determines it needs more info at a later date, or if the customer shows up on a sanctions list.
+PROCESSING | KYC process is in flight and client can check again in the future to see if any further info is needed.
 NEEDS_INFO | More info needs to be provided to finish KYC for this customer.  The `fields` entry is required in this case.
 REJECTED | This customer's KYC has failed and will never succeed.  The `message` must be supplied in this case.
 
 #### Fields
 
-The fields entry is required for the `NEEDS_INFO` status.  Fields should be specified as an object with keys representing field names required, preferably from SEP-9.
+The fields entry is required for the `NEEDS_INFO` status but may be included with any status. Fields should be specified as an object with keys representing the [SEP-9](sep-0009.md) field names required.
 
 Property | Type | Description
 -----|------|------------
 `type` | enum | The data type of the field value. Can be `string`, `binary`, `number`, or `date`
 `description` | string | A human-readable description of this field, especially important if this is not a SEP-9 field.
 `choices` | array | (optional) An array of valid values for this field.
 `optional` | boolean | (optional) A boolean whether this field is required to proceed or not. Defaults to false.
+`status` | string | (optional) One of the values described in [Field Statuses](#field-statuses).
+`error` | string | (optional) The human readable description of why the field is `REJECTED`.
+
+#### Field Statuses
+
+Status | Description
+-------|------------
+ACCEPTED | The field has been validated. When all required fields are accepted, the [Customer Status](#customer-statuses) should also be accepted.
+PROCESSING | The field is being validated. The client can make [GET /customer](#customer-get) requests to check on the result of this validation in the future.
+NOT_PROVIDED | The field has not been sent by the client. The field is required unless the `optional` property is `true`. The [Customer Status](#customer-statuses) should be `NEEDS_INFO` if a field is required but has not been provided.
+REJECTED | The field was in the `PROCESSING` status but did not pass validation. If the client may resubmit this field, the [Customer Status](#customer-statuses) should be `NEEDS_INFO`, otherwise it should be `REJECTED`.
+VERIFICATION_REQUIRED | The field must be verified using the [PUT /customer/verification](#customer-put-verification) endpoint. For example, the `mobile_number` field could be placed in this status until a confirmation code is sent to the customer and passed back to the this endpoint.
```

Comment:
💡 Not needed for this PR, but these statuses would benefit from a diagram showing the entry and exit statuses, and from which states they can transition.